modules/ROOT/pages/prereq-hardware.adoc

```diff
@@ -28,7 +28,7 @@ For production environments, each node in your configuration must have the follo
 | Component |Size|Minimum IOPS|Mount Point|Description
 |HDD 2 xref:prereq-hardware#system-state-directory-device[System State Directory Device] | 100 GB |1500|/var/lib/gravity | Stores system configuration and metadata, including databases and packages. Because package sizes can be large, it is important to estimate the minimum size requirements and allocate enough space as a dedicated device before installation.
 |HDD 3 xref:prereq-hardware#application-data-device[Application Data Device]| 250 GB |1500|/var/lib/data | Stores application configuration and data. The amount of space required should be at least 250 GB, but can vary depending on your specific use case. It is important to estimate the minimum size requirements and allocate enough space as a dedicated device ahead of time.
-|HDD 4 xref:prereq-hardware#docker-device[Docker Device] | 100 GB |3000|Not mounted | Used by Docker’s Device Mapper storage driver. A minimum of 100Gb is required for the Device Mapper directory. Devices with 50Gb or less will experience degraded system performance or might not work at all.
+|HDD 4 xref:prereq-hardware#docker-device[Docker Device] | 100 GB |3000|/var/lib/gravity/planet/docker | Used by Docker’s overlay storage driver. A minimum of 100Gb is required for the overlay directory. Devices with 50Gb or less will experience degraded system performance or might not work at all.
```

Comment:
100 Gb, with space (gigabits, not gigabytes, GB?)
50 Gb
```suggestion
|HDD 4 xref:prereq-hardware#docker-device[Docker Device] | 100 GB |3000|/var/lib/gravity/planet/docker | Used by Docker’s overlay storage driver. A minimum of 100 Gb is required for the overlay directory. Devices with 50 Gb or less experience degraded system performance or might not work at all.
```
Try to avoid tenses other than present/ongoing.